src/sema/types.rs

```diff
@@ -993,10 +993,10 @@ impl Type {
         }
     }
 
-    /// Calculate how much memory we expect this type to use when allocated on the
-    /// stack or on the heap. Depending on the llvm implementation there might be
-    /// padding between elements which is not accounted for.
-    pub fn size_of(&self, ns: &Namespace) -> BigInt {
+    /// Calculate how much memory this type occupies in storage.
+    /// Depending on the llvm implementation there might be padding between elements
+    /// which is not accounted for.
+    pub fn storage_size_of(&self, ns: &Namespace) -> BigInt {
```

Comment:
I think we should be clear this is solana-specific. This should be unused for Substrate/ewasm